src/pages/dataSetsWip/form/AdvancedFormContents.tsx

```diff
@@ -1,11 +1,71 @@
 import i18n from '@dhis2/d2-i18n'
-import React from 'react'
+import { CheckboxField, CheckboxFieldFF } from '@dhis2/ui'
+import React, { useState } from 'react'
+import { Field, useField } from 'react-final-form'
 import {
+    StandardFormField,
     StandardFormSectionDescription,
     StandardFormSectionTitle,
+    ModelTransferField,
 } from '../../../components'
+import { ModelSingleSelectField } from '../../../components/metadataFormControls/ModelSingleSelect'
 import { SectionedFormSection } from '../../../components/sectionedForm'
 
+const NOTIFICATION_RECIPIENTS_QUERY = {
+    resource: 'userGroups',
+}
+
+const NotificationField = () => {
```

Comment:
I haven't put this component in its own file as the advanced contents were not particularly long,